src/electionguard_cli/cli_steps/election_builder_step.py

```diff
@@ -13,19 +14,21 @@ class ElectionBuilderStep(CliStepBase):
     """Responsible for creating a manifest and context for use in an election."""
 
     def build_election_with_key(
-        self,
-        election_inputs: CliElectionInputsBase,
-        joint_key: ElectionJointKey,
+        self, election_inputs: CliElectionInputsBase, joint_key: ElectionJointKey
     ) -> BuildElectionResults:
         return self._build_election(
-            election_inputs, joint_key.joint_public_key, joint_key.commitment_hash
+            election_inputs,
+            joint_key.joint_public_key,
+            joint_key.commitment_hash,
+            None,
```

Comment:
Why not allow a verification URL here? And if so, isn't `build_election_with_key` essentially the same as `_build` ?

Reply:
It's for e2e.  I was opting for simplicity in this PR, but I'll add it.